apis/apiextensions/v1/composition_types.go

```diff
@@ -378,6 +378,8 @@ type ContainerFunctionRunner struct {
 // +genclient:nonNamespaced
 
 // A Composition specifies how a composite resource should be composed.
+// +kubebuilder:printcolumn:name="KIND",type="string",JSONPath=".spec.compositeTypeRef.kind"
+// +kubebuilder:printcolumn:name="APIVERSION",type="string",JSONPath=".spec.compositeTypeRef.apiVersion"
```

Comment:
Would it make sense to clarify that these are the kind of compatible XR somehow? Maybe `XR-KIND`?

Reply:
I was trying to think of something to do that - should have been obvious.  I'll make the change.  Thanks!